lib/instrumentation/transaction.js

```diff
@@ -161,6 +161,19 @@ Transaction.prototype.toJSON = function () {
     }
   }
 
+  // add sample_rate to transaction
+  // https://github.com/elastic/apm/blob/master/specs/agents/tracing-sampling.md
+  // Only set sample_rate on transaction payload if a valid trace state
+  // variable is set.
+  //
+  // "If there is no tracestate or no valid es entry with an s attribute,
+  //  then the agent must omit sample_rate from non-root transactions and
+  //  their spans."
+  const sampleRate = this.sampleRate
+  if (sampleRate || sampleRate === 0) {
```

Comment:
nit: Perhaps use `if (sampleRate !== null)` given that the sampleRate getting returns `null` for an invalid value.

Reply:
✅